workflows/differentialabundance.nf

```diff
@@ -113,149 +125,219 @@ workflow DIFFERENTIALABUNDANCE {
     // Set up some basic variables
     ch_versions = Channel.empty()
 
-    // If we have affy array data in the form of CEL files we'll be deriving
-    // matrix and annotation from them
-
-    if (params.study_type == 'affy_array'){
+    // Check first if maxquant, as in that case the entire VALIDATE block can be skipped
```

Comment:
Indeed, but the maxquant matrices have a slightly different structure compared to the rnaseq stuff so the validator does not work for them ATM. I can check how it needs to be changed so it can be kept for the Px path as well

Reply:
Okay, the validator worked actually with the proteus output. The checks are now kind of double but nvm